### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/org/zaproxy/zap/extension/websocket/resources/help_pt_BR/contents/options.html [hunk 1]
```diff
@@ -12,14 +12,14 @@ <h1>Opções específicas de WebSocket</h1>
 Esta tela permite que você configure as opções de WebSockets:
 
 <h3>Encaminhar tudo</h3>
-Se você não está interessado na comunicação de WebSockets, mas deseja permitir que essas mensagens trafeguem pelo ZAP, deve habilitar esta opção.
+Se você não está interessado na comunicação via WebSockets, mas deseja permitir que essas mensagens trafeguem pelo ZAP, deve habilitar esta opção.
 Como resultado nenhuma mensagem será armazenada no banco de dados de sessão. Além disso, nenhuma mensagem aparecerá na guia de WebSockets. Essa configuração pode ser útil se você tem que lidar com conexões de WebSocket críticas em desempenho, e não se interessa pelo que está sendo enviado.
 <br><br>
-Se você quiser ignorar um canal específico, confira as Propriedades de Sessão <i>específicas de Websocket</i>.
+Se você quiser ignorar um canal específico, confira as Propriedades de Sessão <i>Propriedades da Sessão</i>.
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/websocket/resources/help_pt_BR/contents/options.html]: Does it seem strange that websocket seems translated here but not beside the change above?

Comment (thc202) [reply src/org/zaproxy/zap/extension/websocket/resources/help_pt_BR/contents/options.html]: Yeah, this is not correct, I'll revert.

**Comment by thc202** [comment]
Closing, per review comment.

**Comment by thc202** [comment]
For the record, that change is now reverted.

**Comment by kingthorin** [comment]
TY!

**Pull request closed without merging**